packages/headless/src/features/product-listing/product-listing-analytics.actions.ts

```diff
@@ -0,0 +1,32 @@
+import {Result} from '../..';
```

Comment:
we don't normally end file names with `.actions.ts`, it looks like it was a typo in the case of `product-recommendations`.

The name should be `product-listing-analytics-actions.ts`.

Reply:
Thanks for letting me know !